tests/test_delineateit.py

```diff
@@ -188,12 +230,13 @@ def test_point_snapping(self):
         snapped_points_layer = snapped_points_vector.GetLayer()
 
         # snapped layer will include 3 valid points and one polygon.
-        self.assertEqual(4, snapped_points_layer.GetFeatureCount())
+        self.assertEqual(5, snapped_points_layer.GetFeatureCount())

```

Comment:
Need to update this comment?

Reply:
Yep!  I added in a missing assertion while I was in there as well.  abc0e171